### PR TITLE
feat: add gift link

### PR DESCRIPTION
**Pull request by ChefJerry**
---

## PR-Codex overview
The focus of this PR is to add a `WBNBGiftLink` component to display a gift link for WBNB staking campaigns in the `AprCell` component.

### Detailed summary
- Added `WBNBGiftLink` component for WBNB staking campaigns
- Conditionally render `WBNBGiftLink` based on selected pool token
- Utilized `bscTokens` from `@pancakeswap/tokens`
- Updated rendering logic based on selected pool token

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/views/FixedStaking/components/AprCell.tsx [hunk 1]
```diff
@@ -17,6 +32,7 @@ export default function AprCell({
 }) {
   return selectedPeriodIndex === null || !selectedPool ? (
     <Flex alignItems="center">
+      {pool.token.chainId === bsc.id && pool.token.symbol === 'WBNB' ? <WBNBGiftLink /> : null}
```
Comment (Chef-Yogi) [inline apps/web/src/views/FixedStaking/components/AprCell.tsx:35]: how about use the token equal method here?

Comment (Chef-Yogi) [reply apps/web/src/views/FixedStaking/components/AprCell.tsx:35]: pool.token.equals(bscTokens.bnb)
something like this

Comment (ChefJerry) [reply apps/web/src/views/FixedStaking/components/AprCell.tsx:35]: you mean import bscTokens?

Comment (Chef-Yogi) [reply apps/web/src/views/FixedStaking/components/AprCell.tsx:35]: yes, do you think it's a good idea? 
if it's not, it's ok, just found the method usage from the Chef Jackson's previous PR

**Review by Chef-Yogi — APPROVED** [review]

**Review by Chef-Yogi — APPROVED** [review]

**Pull request merged**